zetaclient/zetacore/query.go

```diff
@@ -194,6 +205,7 @@ func (c *Client) GetAbortedZetaAmount() (string, error) {
 	return resp.AbortedZetaAmount, nil
 }
 
+// GetGenesisSupply returns the genesis supply
 func (c *Client) GetGenesisSupply() (sdkmath.Int, error) {
 	tmURL := fmt.Sprintf("http://%s", c.cfg.ChainRPC)
 	s, err := tmhttp.New(tmURL, "/websocket")
```

Comment:
Should we save these two values to config ?

Reply:
You mean `/websocket`? I don't think so